src/deepak/A15_ArrayOperations.java

```diff
@@ -0,0 +1,110 @@
+/*Assignment - 15: 30th Dec'2021
+
+Write a program to satisfy below requirement. Make sure method should take Array as parameter. 
+Program 1 : return maximum number from give array.
+Program 2 : return minimum number from given array.
+Program 3: return count of positive numbers from given array.
+sample input : int[] arr = {12,-11,99,11,23,-44}
+output : 4
+
+Program 4 : return sum of positive numbers from given array.
+sample input : int[] arr = {10,20,-33,-55,2};
+output : 32
+
+Program 5 : print sum of all even positive numbers from given array.
+sample input : int[] arr = {10,13, -14, -20, 22, 3};
+output : 32
+hint : 10 + 22 = 32
+
+Program 6 : return name having maximum length from given array.
+input : String[] arr = {"Karan", "Shweta", "Pallavi", "Abhishek", "Sagar"};
+output : Abhishek
+*/
+package deepak;
+
+public class A15_ArrayOperations {
+
+	int getMaxNumInArray(int[] input) {
+		int max = input[0];
+		for (int index = 0; index < input.length; index++) {
```

Comment:
If you are taking the 0th index as max number then you can start the loop from index 1 as well, same for other methods